### DISCUSSION TITLE
画像を隠すボタンについて

### DISCUSSION THREAD

**Discussion by ikakonbu** [misskey-dev/misskey#11009] [Polls]
### 概要
以前のバージョンでは右上の非表示ボタンから１クリックで画像を非表示にできたが、
現在の最新バージョンでは画像表示時に、右下の･･･ボタンをクリックし、ポップアップの「隠す」ボタンを押さないと画像を非表示に出来ない。
この点について、他の方がどう考えているのか気になる。
![image](https://github.com/misskey-dev/misskey/assets/82440954/ed993671-d227-4499-bb0f-f4bf89528a22)

### 私が求めていること
NSFW設定されたメディアを誤ってクリックしてしまい、すぐに非表示にできないので、可能であれば以前のようにワンクリックで非表示になるように戻してほしい。

### 別途要望
また、どちらにせよボタンが小さく、画像がタップされて全画面表示になってしまうことがよくあったので、非表示ボタンを大きくする、ボタン押下判定を広げるなどしてほしい

**Comment by EdamAme-x** [comment]
確かに以前の仕様のほうが良いと思いますね
現状　画像に関する操作のパレットとして試験的にそうなっているのだと思います。
